Training Plan/BeginningTraining.md

```diff
@@ -0,0 +1,133 @@
+# Basic training Schedule
+
+## First Meeting:
+ - [ ] Install software
+  - [WPI VS Code](https://github.com/wpilibsuite/allwpilib/releases)
+  - FRC Update (Driver Station)
+  - [GitKraken](https://www.gitkraken.com/)
+  - [Git](https://git-scm.com/)
+  - [Radio programming tool](https://docs.wpilib.org/en/latest/docs/software/getting-started/radio-programming.html) (Advanced Students)
+ - [ ] Demo of basic driving robot code
+  - Start with OI and subsystem created, including default command defined
```

Comment:
In commands-experimental (in the namespace `edu.wpi.first.wpilibj.frc2.command`), default commands are optionally set in the constructor, instead of having to override a particular method. I assume we plan on teaching the new library?